stellar/docs/link-token.md

```diff
@@ -129,19 +160,64 @@ Register the token on the source chain (Stellar):
 ts-node stellar/its.js register-custom-token <salt> <tokenAddress> <tokenManagerType>
 ```
 
-### Step 4: Link Token
+### Step 4: Verify Token Metadata Registration
+
+Before linking tokens, verify that token metadata is registered on ITS Hub for both chains:
+
+```bash
+# Verify source token metadata (Stellar)
+axelard query wasm contract-state smart $ITS_HUB_ADDRESS '{"custom_tokens":{"chain":"<sourceChain>","token_address":"<tokenAddress>"}}' --output json
+
+# Verify destination token metadata (EVM)
+axelard query wasm contract-state smart $ITS_HUB_ADDRESS '{"custom_tokens":{"chain":"<destinationChain>","token_address":"<destinationTokenAddress>"}}' --output json
+```
```

Comment:
do this in the script, instead of making the user perform it. you can grep a simple example of how to do a cosmwasm query in the scripts